packages/terra-date-picker/src/terra-dev-site/test/date-picker/DisabledDatePickerField.test.jsx

```diff
@@ -12,7 +12,6 @@ const DatePickerDisabledDate = () => (
       name="date-input"
       selectedDate="2017-04-01"
       datePickerId="disabled-date-picker"
-      disabled
```

Comment:
Instead of removing `disabled` from this test to use for the demo, could you use https://github.com/cerner/terra-framework/blob/main/packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerField.test.jsx instead of possible create a new test?

Reply:
Created new test `DatePickerFieldDefault.test.jsx`.